src/snapshot/SnapshotMatcher.lua

```diff
@@ -0,0 +1,101 @@
+local ReplicatedStorage = game:GetService("ReplicatedStorage")
+
+local Markers = require(script.Parent.Serialize.Markers)
+local Serialize = require(script.Parent.Serialize)
+local deepEqual = require(script.Parent.Parent.deepEqual)
+local ElementKind = require(script.Parent.Parent.ElementKind)
+
+local SnapshotFolderName = "RoactSnapshots"
+local SnapshotFolder = ReplicatedStorage:FindFirstChild(SnapshotFolderName)
+
+local SnapshotMatcher = {}
+local SnapshotMetatable = {
+	__index = SnapshotMatcher,
+	__tostring = function(snapshot)
```

Comment:
should Snapshot.lua return tables with a metatable like this instead of the matcher? It feels more intuitive to be able to call tostring on the snapshot itself instead of the matcher

Reply:
Yes I removed it and I added a `getSnapshotString` to ShallowWrapper and SnapshotMatcher